monkey/monkey_island/Pipfile

```diff
@@ -5,23 +5,18 @@ name = "pypi"
 
 [packages]
 pyinstaller = "==3.6"
-awscli = "==1.20.44"
 bcrypt = "==3.2.0"
 boto3 = "==1.18.44"
 botocore = "==1.21.44"
-cffi = ">=1.8,!=1.11.3"
 dpath = ">=2.0.5"
 gevent = ">=20.9.0"
 ipaddress = ">=1.0.23"
 jsonschema = "==3.2.0"
 mongoengine = "==0.20"
 netifaces = ">=0.10.9"
 pycryptodome = "==3.9.8"
-python-dateutil = "<3.0.0,>=2.1"
```

Comment:
This is used in a bunch of places. Just search for `import dateutil`.

Reply:
It's annoying when packages don't use the same name. Also, I got no errors because it's a third-party dependency.